src/features/Account/ManageAccountForm.tsx

```diff
@@ -296,26 +304,39 @@ const ManageAccountForm: React.FC<IManageAccountProps> = (props) => {
             value={fp.values.gender}
           />
           <ErrorMessage component={FormikElements.Error} name="pronoun" />
-          <FastField
-            name={'dietaryRestrictions'}
-            isMulti={true}
-            label={CONSTANTS.DIETARY_RESTRICTIONS_LABEL}
-            placeholder={DietaryRestriction.NONE}
-            component={FormikElements.Select}
-            options={getOptionsFromEnum(DietaryRestriction)}
-            required={true}
-            value={fp.values.dietaryRestrictions}
-          />
-          <ErrorMessage
-            component={FormikElements.Error}
-            name="dietaryRestrictions"
-          />
+
+          {
+            // Hide dietary restrictions on account creation for 2020 year - don't need because online only hackathon
+            // Add back in for future years (hopefully) :(
+            props.mode !== ManageAccountModes.CREATE && <>
```

Comment:
Two questions:
1. Have you confirmed that by removing this field, account creation still works?
2. Consider refactoring this logic to some function `IncludeDietaryRestrictions`, that returns true if we want to ask for dietary restrictions, and false otherwise. This way, we can potentially have a checkbox in the settings schema that allows for a 'virtual hackathon', and disables all in-person hackathon features. After we have that, we can reroute `IncludeDietaryRestrictions` to check the settings schema or something.

Reply:
1. Yeah, have confirmed account creation still works (dietary restriction defaults to `[]`). Though, on second thought, would be better to set it to something like `["Unknown"]` so we can reconcile next year. Otherwise, with accounts persisting through years, we don't want to end up with someone next year not having their dietary restrictions recorded just because they first signed up this year

2. Yeah, this is a really good idea. The same logic can be applied to two other fields in application (tshirt size and travel disbursement request). There is a good chance other edge cases for remote hackathons will arise too (e.g. hiding travel tab in a non-hardcoded way).


Tshirt size and travel are safe to just default to `"S"` and `0` for all applications this year, as they don't carry through to next hackathon. Dietary restriction model be modified to include "Unknown" as an option, to allow for reconciliation next year